### PR TITLE
fix(proposal): display correct information for the "Fulfillment date" field in Invoice Details ⛽

### PR DESCRIPTION
**Pull request by radist2s**
Update the "_Fulfillment date_" field in the Invoice Details component to display the correct date format. The previous implementation displayed contact details instead of the fulfillment date.

[📷 Fixed UI screenshot](https://github.com/team-monite/monite-sdk/assets/725645/e2b5517b-a0e3-46a8-9373-e141df3bfe44)

### CODE REVIEW

File: packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/sections/PreviewDetailsSection.tsx [hunk 1]
```diff
@@ -31,9 +32,12 @@ export const PreviewDetailsSection = ({
           },
           {
             label: t(i18n)`Fulfillment date`,
-            value:
```
Comment (saladNights) [inline packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/sections/PreviewDetailsSection.tsx:34]: Why did you remove the counterpart info? There is no Jira task, so it's not obvious.

**Review by MadMed677 — APPROVED** [review]

File: packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/sections/PreviewDetailsSection.tsx (hunk 1 above)
Comment (MadMed677) [reply packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/sections/PreviewDetailsSection.tsx:34]: The check wasn't correct. I think I just copy-pasted it for fulfillment date so the check for counterparts wasn't needed here

**Review by saladNights — APPROVED** [review]
